test/solver_tests/dissimilarity_functions_test.py

```diff
@@ -253,6 +253,21 @@ def test_weighted_hamming_similarity_all_missing(self):
 
         self.assertEqual(similarity, 0)
 
+    def test_hamming_distance(self):
 
+        distance = dissimilarity_functions.hamming_distance(self.s1, self.s2)
+
+        self.assertEqual(distance, 3)
+
+    def test_hamming_distance_ignore_missing(self):
+
+        distance = dissimilarity_functions.hamming_distance(self.s1, self.s2, ignore_missing_state=True)
+
+        self.assertEqual(distance, 2)
+
+        distance = dissimilarity_functions.hamming_distance(self.s1, self.all_missing, ignore_missing_state=True)
+
+        self.assertEqual(distance, 0)
+        
```

Comment:
Pretty long line here. Not sure if we're supposed to run `black` on tests, but if we do, this should be broken up by running it. Otherwise, feel free to resolve and ignore.

Reply:
I've run black on this file now.